library/src/lib/localizator-editor/localization-editor-item/localization-editor-item.component.ts

```diff
@@ -0,0 +1,46 @@
+import { Component, ContentChild, Input, OnInit, TemplateRef } from '@angular/core';
+import { LocalizationEditorInputDirective, LocalizationEditorLabel } from '../localization-editor.directives';
+
+/**
+ *  Component that represents field with add-on. It is made to be inside popover.
```

Comment:
Component that represents a field with an add-on. 

the popover info can be omitted I think.